CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## 2.29.0-rc.0 / 2021-07-29
+
+* [CHANGE] Promote `--storage.tsdb.allow-overlapping-blocks` flag to stable. #9117
+* [CHANGE] Promote `--storage.tsdb.retention.size` flag to stable. #9004
+* [FEATURE] Add Kuma service discovery. #8844
+* [FEATURE] Add `present_over_time` PromQL function. #9097
+* [FEATURE] Allow configuring exemplar storage via file and make it reloadable. #8974
```

Comment:
I clicked over to see how the "storage via file" worked, only to realise it's "configuring via file".
So you might want to reword for clarity.